src/HDInsight/HDInsight/ClusterConfigurationUtils.cs

```diff
@@ -74,7 +74,7 @@ public static string GetResourceGroupFromClusterId(string clusterId)
                         storageRootPath: storageRootPath,
                         applicationId: appId,
                         tenantId: tenantId,
-                        certificateContents: Convert.FromBase64String(certContents ?? ""),
```

Comment:
In the code here is the only way to set `AzureHDInsightDataLakeDefaultStorageAccount.certificateContents`. This will make `certificateContents` be null forever. Is this you expect? Would you please add a test case for this?

Reply:
> In the code here is the only way to set `AzureHDInsightDataLakeDefaultStorageAccount.certificateContents`. This will make `certificateContents` be null forever. Is this you expect? Would you please add a test case for this?

Hi, @wyunchi-ms , thanks for your reminder, after discuss with @idear1203 this is exactly we want. As for test case, we have did teste case offline.